app/build.gradle

```diff
@@ -82,19 +81,13 @@ android {
             applicationIdSuffix ".fenix.debug"
             resValue "bool", "IS_DEBUG", "true"
             pseudoLocalesEnabled true
-            def deepLinkSchemeValue = "fenix-dev"
-            buildConfigField "String", "DEEP_LINK_SCHEME", "\"$deepLinkSchemeValue\""
-            manifestPlaceholders = [
-                    "deepLinkScheme": deepLinkSchemeValue,
```

Comment:
Please don't remove the `deepLinkScheme` placeholder for any of the variants. We rely on this for identifying deeplinks across variants.